packages/devtools_app/lib/src/inspector/layout_explorer/flex/flex.dart

```diff
@@ -452,6 +364,149 @@ class _FlexLayoutExplorerWidgetState extends LayoutExplorerWidgetState<
   }
 }
 
+class VisualizeFlexChildren extends StatefulWidget {
+  const VisualizeFlexChildren({
+    Key key,
+    @required this.state,
```

Comment:
I don't think this is new code, but it seems strange that we are passing a state around. Is this needed? from what I can tell it isn't used anywhere in this widget.

Reply:
it is needed and I agree this is an anti-pattern. I looked for a cheap way to avoid it for this CL but it would have been more of a refactor than I wanted to add here. Filed https://github.com/flutter/devtools/issues/2701 to track.